### ISSUE TITLE
About the novelty of the model

### ISSUE THREAD

**Issue by Levishery** [wengong-jin/hgraph2graph#13]
Could you please provide the Novelty(the fraction of unique valid generated molecules not present in the training set) of your model? It seems that this important evaluation metric is missing in the paper.
